packages/client/components/TeamPrompt/TeamPromptDrawer.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {commitLocalUpdate, useFragment} from 'react-relay'
+import useAtmosphere from '~/hooks/useAtmosphere'
+import {TeamPromptDrawer_meeting$key} from '~/__generated__/TeamPromptDrawer_meeting.graphql'
+import {desktopSidebarShadow} from '../../styles/elevation'
+import {BezierCurve, DiscussionThreadEnum, ZIndex} from '../../types/constEnums'
+import ResponsiveDashSidebar from '../ResponsiveDashSidebar'
+import TeamPromptDiscussionDrawer from './TeamPromptDiscussionDrawer'
+import TeamPromptWorkDrawerRoot from './TeamPromptWorkDrawerRoot'
+
+const Drawer = styled('div')<{isDesktop: boolean; isOpen: boolean}>(({isDesktop, isOpen}) => ({
+  boxShadow: isDesktop ? desktopSidebarShadow : undefined,
+  backgroundColor: '#FFFFFF',
+  display: 'flex',
+  flex: 1,
+  flexDirection: 'column',
+  justifyContent: 'stretch',
+  overflow: 'hidden',
+  position: isDesktop ? 'fixed' : 'static',
+  bottom: 0,
+  top: 0,
+  right: isDesktop ? 0 : undefined,
+  userSelect: isDesktop ? undefined : 'none',
+  transition: `all 200ms ${BezierCurve.DECELERATE}`,
+  transform: `translateX(${isOpen ? 0 : DiscussionThreadEnum.WIDTH}px)`,
+  width: DiscussionThreadEnum.WIDTH,
```

Comment:
+1 I think `DiscussionThreadEnum.WIDTH` is too big for mobile displays, which is why on mobile we have to stack the discussion thread below the card area in retros (I see it's been this width for standups, just haven't poked around til now!). Could we reduce it to the 256-296px range? That way we leave enough of the scrim as a click target (I think 16px of the scrim is for the peek, so we really only have a few pixels where we can click to exit). We could also make the width dependent on device! 

<img width="281" alt="Screenshot 2023-08-22 at 11 42 54 AM" src="https://github.com/ParabolInc/parabol/assets/5514175/52b3520e-43bf-4319-be69-5c590e421bb5">

Reply:
I went a slightly different direction here - the actual CSS is a bit more complicated due to how the sidebar components are structured, but I've changed the effective behavior to something like `width: DiscussionThreadEnum.WIDTH, max-width: 100vw`, so the width will start shrinking once the viewport width is <`DiscussionThreadEnum.WIDTH`.

I could've preserved a few pixels for "tap outside to close" behavior, but between the close button and "swipe right to close" behavior I think it's fine to allow this to go up to full-width on those smallest screens, but let me know what you think!

Reply:
other 2 fixes look good! This one is pretty much the same for iPhone 12/Pixel 5, which both have ~390px width. If you don't wanna shrink the width you could make it go full screen & just say screw the scrim. main tension is why show the scrim when my fingers are too fat to click it? 

For rough numbers:
- minimum clickable width: 48px (google material design suggestion)
- peek handler:  20px
- target width: ~390px (I think we actually target a min of 360? but that was from years ago, devices have gotten bigger)
- 390 - 20 - 48 = 322 max width

In other parts we try to keep the right drawer at 256, but with cards in it we may have to go bigger https://github.com/ParabolInc/parabol/blob/69cad570a630f877f2f01aea79cd7fdd1431fb39/packages/client/types/constEnums.ts#L331

Reply:
@mattkrick  Ahh gotcha. I prefer to grow the width over shrink it here, so I changed this to just go full-screen for breakpoints under `FUZZY_TABLET` (700px).

Reply:
LOVE IT!